kompose.spec

```diff
@@ -190,11 +185,15 @@ export GOPATH=$(pwd):%{gopath}
 export GOPATH=$(pwd):$(pwd)/vendor:%{gopath}
```

Comment:
That is true. You can drop the ``$(pwd)/vendor``. I have [updated](https://github.com/gofed/gofed/commit/3ca8063442dfcaedc8a70a3612a420e1b4916ac7) the generator and removed vendor from GOPATH.